argobots.spec

```diff
@@ -53,6 +58,7 @@ Development files for the argobots library.
 
 %prep
 %autosetup -p1
+%patch0 -p1
```

Comment:
You don't need to specify this if `%autosetup` is being used, as it is on line 60.
```suggestion
```

Reply:
ok